10981.0.txt

```diff
@@ -0,0 +1,37 @@
+@0
+
+
+@1
+##DOUBLER
+
+#STORE INT IN ACC
+MOV UP, ACC
+
+#ADD INT IN ACC
+#TO ITSELF
```

Comment:
Good effort at cramming meaningful comments into the space allowed by the game, but I'm glad you abandoned this for later levels!